contributing.md

```diff
@@ -0,0 +1,6 @@
+![](https://raw.githubusercontent.com/checkstyle/resources/master/img/checkstyle-logos/checkstyle-logo-260x99.png)
+ # Contributing Guide
+ ## Please follow the below links in order to start contributing to the checkstyle project
+#### [Beginning Development](http://checkstyle.sourceforge.net/beginning_development.html)
+#### [Importing checkstyle project to an IDE](http://checkstyle.sourceforge.net/idea.html)
```

Comment:
why only IDEA link ? we have links to 3 major IDEs

Reply:
@romani On it. Would do the changes suggested above